js/kontent.js

```diff
@@ -0,0 +1,63 @@
+const projectId = "99b0b3b0-4838-0051-3d57-8af72f55e8a0";
+const apiBlogItems = `https://deliver.kontent.ai/${projectId}/items`;
+const apiBlogItem = `https://deliver.kontent.ai/${projectId}/items/`;
+const apiTaxonomies = `https://deliver.kontent.ai/${projectId}/taxonomies/`;
+const pageSize = 10;
+
+class URLBuilder {
+    constructor(baseURL) {
+        this.singleEndpoint = null;
+        this.includeTotalCount = true;
+        this.limit = null;
+        this.orderByElementProp = null;
+        this.blogTagsContains = null;
+        this.baseURL = new URL(baseURL + (this.singleEndpoint || ""));
```

Comment:
`this.singleEndpoint` will always be `null`

Reply:
Except when it's set

Reply:
There's no way to set it during the constructor when this code runs.

Reply:
Now, I see what you meant...